ipatests/prci_definitions/gating.yaml

```diff
@@ -243,3 +243,15 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
+  fedora-29/test_automount_locations:
```

Comment:
automount is neither super critical nor likely to fail. IMHO it's sufficient to test the feature in the nightly tests.